### ISSUE TITLE
Unable to Display Tiles in ArcGIS Pro of TPKX File facing issue with bundle files

### ISSUE THREAD

**Issue by AviralPadlia** [lucianpls/MaBaker#1]
Hello,

I hope this message finds you well. I'm encountering an issue while attempting to convert a GeoTIFF image to TPKX format using the raster-tiles-compactcache repository.

Steps to Reproduce:

    Converted the GeoTIFF image into tiles using gdal2tiles.
    Processed these tiles using the "vundler.py" program from the "raster-tiles-compactcache" repository.
    Generated item.json, root.json, and thumbnail files and assembled them into a folder in the TPKX format.

Expected Behavior:
When attempting to visualize the TPKX output in ArcGIS pro, I should see the tiles.

Actual Behavior:
I can’t see tiles on map in ArcGIS pro when I add tpkx file, But it zooms to right location

Additional Information:
    I suspect there might be an issue during the creation of bundle files using vundler.py.
    In my search for solutions, I came across the mabaker repository, which also deals with creating bundle files.
    I'm reaching out to inquire if there's a particular part of your codebase that could assist in generating these bundles correctly or if there are any insights you might provide to resolve this issue.

Your guidance or suggestions on rectifying this problem would be greatly appreciated.

Thank you for your time and assistance in advance.

**Comment by lucianpls** [comment]
Try debugging step by step

**Comment by AviralPadlia** [comment]
I found the issue! The problem is, my GeoTiff image contains transparent area cause the maps they show have irregular boundaries. So the tiles which contains only transparent pixels and no RGB pixels, aren't showing up in the bundle files as a blank image and leading to fail all the tiles to be seen in ArcGIS pro.

For example, think of an image showing a circular map. The corner don't have any RGB pixels, so tiles generated from that part wont be added in the bundle file.

The thing is, these transparent parts aren't getting included in the bundle files for ArcGIS Pro. If there's even one little pixel in a tile, it's added, but if there's none, it gets left out. I also tried running mabaker.py but transparent tiles are not generated in it as well.

Your guidance or suggestions on rectifying this problem would be greatly appreciated.

Thank you for your time and assistance in advance
